webpack/src/extensions/host/HostBootdiskButtons.js

```diff
@@ -40,6 +59,7 @@ const HostBootdiskButtons = () => {
         href={foremanUrl(action.link)}
         isDisabled={action.disabled}
         description={action.description}
+        icon={iconComponent(action?.icon)}
```

Comment:
since we know how to handle even the `undefined` case in `iconComponent`
and `action` is always an object, it would be ok to omit the `?.`
```suggestion
        icon={iconComponent(action.icon)}
```